### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/cms": "^4",
-        "silverstripe/framework": "^4.11",
-        "silverstripe/admin": "^1",
-        "silverstripe/versioned": "^1",
-        "symfony/yaml": "^3 || ^4"
+        "php": "^8.1",
+        "silverstripe/cms": "^5",
+        "silverstripe/framework": "^5",
```
Comment (michalkleiner) [inline composer.json:20]: Do we need to require the framework when we already require the cms module? CMS also already requires versioned and admin.

Comment (dhensby) [reply composer.json:20]: You should always explicitly declare your dependencies. If you use framework's classes/APIs you should be declaring it as a dependency. There is no obligation for any of your dependencies (ie: CMS) to always include framework. So if a packaged needs a dep, it should declare it.

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Failing unit tests

**Comment by sabina-talipova** [comment]
Tests passed

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**